tests/src/test/scala/common/rest/WskRest.scala

```diff
@@ -138,8 +141,11 @@ trait ListOrGetFromCollectionRest extends BaseListOrGetFromCollection {
                     expectedExitCode: Int = OK.intValue)(implicit wp: WskProps): RestResult = {
 
     val entPath = namespace map { ns =>
-      val (ns, name) = getNamespaceEntityName(resolve(namespace))
-      Path(s"$basePath/namespaces/$ns/$noun/$name/")
+      val (nspace, name) = getNamespaceEntityName(resolve(namespace))
+      name match {
```

Comment:
`if (name.isEmpty) ... else ...` is clearer imo.

Reply:
Done